### DISCUSSION TITLE
Arco desgin Vue版本怎么使用cdn加速？

### DISCUSSION THREAD

**Discussion by houlovegu** [arco-design/arco-design-vue#2722] [Q&A]
因为是海外项目，部署在国外节点，加载arco.js耗时要20秒 ，我看react版本有cdn配置，请问vue版本和react配置一样么？

**Comment by yinkaihui** [comment]
一样的

js: https://unpkg.com/@arco-design/web-vue@latest/dist/arco-vue.min.js
图标: https://unpkg.com/@arco-design/web-vue@latest/dist/arco-vue-icon.min.js
